### PR TITLE
Add create message functionality

### PR DESCRIPTION
**Pull request by saadshahd**

### CODE REVIEW

File: src/app/chat.js [hunk 1]
```diff
@@ -7,11 +7,13 @@ angular
     templateUrl: 'app/chat.html'
   });
 
-function ChatCtrl(chatService) {
+function ChatCtrl($scope, chatService, userService, Upload) {
```
Comment (joehany) [inline src/app/chat.js:10]: Why $scope is injected here without using it

File: src/app/chat.js [hunk 2]
```diff
@@ -20,7 +22,37 @@ function ChatCtrl(chatService) {
...
+      isMine: true,
+      participant: Object.assign({}, userService.get())
+    });
+  }
+
+  function addMessage(message) {
+    chat.data.messages.push(message);
+  }
+
+  const createMessage = _.flow(generateMessage, addMessage);
```
Comment (joehany) [inline src/app/chat.js:40]: Move to the top

Comment (saadshahd) [inline src/app/chat.js:40]: باستخدم 
guide open minded
 شوية

مش بتمنع دا :kiss:

Comment (saadshahd) [inline src/app/chat.js:40]: It's called `xo`

File: src/app/chat.js (hunk 1 above)
Comment (saadshahd) [inline src/app/chat.js:10]: :+1: I'll remove it :smile:

**Pull request merged**